tests/test_interpreters.py

```diff
@@ -6,24 +6,20 @@
 import py
 import pytest
 
+import tox
 from tox.config import get_plugin_manager
-from tox.interpreters import ExecFailed
-from tox.interpreters import InterpreterInfo
-from tox.interpreters import Interpreters
-from tox.interpreters import NoInterpreterInfo
-from tox.interpreters import pyinfo
-from tox.interpreters import run_and_get_interpreter_info
-from tox.interpreters import sitepackagesdir
-from tox.interpreters import tox_get_python_executable
-
-
-@pytest.fixture
-def interpreters():
+from tox.interpreters import (
+    ExecFailed, InterpreterInfo, Interpreters, NoInterpreterInfo, pyinfo,
+    run_and_get_interpreter_info, sitepackagesdir, tox_get_python_executable)
+
+
+@pytest.fixture(name="interpreters")
+def create_interpreters_instance():
     pm = get_plugin_manager()
     return Interpreters(hook=pm.hook)
 
 
-@pytest.mark.skipif("sys.platform != 'win32'")
+@pytest.mark.skipif(not tox.INFO.IS_WIN, reason="not for windows")
```

Comment:
shouldn't the message explain why this is not for windows, I mean the expression already says not for windows; I think would be nicer to explain here why this cannot run/succeed on windows

Reply:
IMO opinion it is completely unnecessaey to say anything as reason, but if you use a boolean expression as condiition it is enforced by pytest. I'd say markers are better for this anyway, so I'll do it that way.